src/app/shared/constants/csv.constants.ts

```diff
@@ -43,17 +34,16 @@ export class CSVConstants {
 
     /* (priorities) */
 
-    StudentComments: 'Comments',
-    SupervisorRating: 'supervisorRating',
-    TutorComments: 'CommentsTutor',
-    IsPinned: 'IsPinned',
+    StudentComments: 'studentComments',
+    SupervisorAssessment: 'supervisorAssessment',
+    TutorComments: 'tutorComments',
+    IsPinned: 'isPinned',
```

Comment:
Should a student being pinned to a team the same thing as having been assigned to a team? Because I think the CSV currently treats an assigned team and being pinned as separate states, unless I misunderstood something. I ask because I'm wondering whether during the matching, once the automatic distribution has placed all students in teams, you'd like to be able to pin _some_ students, export the intermediate result to PROMPT and then come back and do another distribution later, keeping pinned students in their teams. If we only used one value `pinnedToTeam` if I understand correctly all students would be "pinned" after a distribution by the solver.

Reply:
So do we need to add 2 attributes to the spec for a `Student` object, a `pinnedToTeam: <team>` as well as `teamName: <team>`?

Reply:
Why would you need both? I think `pinnedToTeam` is enough to implement the pinning feature. Why would you need the second one?

Reply:
If I'm not mistaken our spec currently doesn't have a `project` field for the `Student` object, because I guess we've so far only considered the initial situation where PROMPT sends the data on all students for the first time (so no students have been assigned to a project yet, hence there being no need for such a field). But if we want to allow for exporting of an "intermediate" matching, where some students have been assigned a project but some are still free agents, then we would need a `project` field so that when the data is re-imported from PROMPT, TEASE knows which students are already assigned to a project (this is what the CSV currently already supports with the `teamName` column, in the example CSV there are three students that are already in the `BMW` team and this is reflected in the UI upon importing). These three students are however not pinned, and thus don't necessarily stay in the `BMW` team upon distribution. Hence why I mention needing two fields, but I could be misunderstanding something here about the intermediate export etc.

Reply:
Went ahead and added `isPinned` as well as `team` to the spec in [9998403](https://github.com/ls1intum/team-allocator/pull/43/commits/999840375957cc6dd84ef47ed14d790fa76c6cec).